packages/ibm-products/src/components/Datagrid/Extensions/Filtering/Panel.stories.jsx

```diff
@@ -24,7 +26,7 @@ export default {
   parameters: {
     styles,
     docs: { page: DocsPage },
-    layout: 'fullscreen',
+    layout: 'padded',
```

Comment:
Guessing we want to keep these `fullscreen`?

Reply:
Yep, that was leftover from the tab example. I'll revert to `fullscreen`.